### PR TITLE
Don't use config dir in root

### PR DESCRIPTION
**Pull request by mnagy**
cc @soltysh

### CODE REVIEW

File: lib/vagrant-openshift/action/install_openshift.rb [hunk 1]
```diff
@@ -41,7 +41,7 @@ def call(env)
 cat > /etc/profile.d/openshift.sh <<DELIM
 export GOPATH=/data
 export PATH=$ORIGIN_PATH/_output/etcd/bin:$ORIGIN_PATH/_output/local/go/bin/:$GOPATH/bin:$PATH
-export OPENSHIFTCONFIG=/openshift.local.config/master/admin.kubeconfig
+export OPENSHIFTCONFIG=openshift.local.config/master/admin.kubeconfig
```
Comment (mfojtik) [inline lib/vagrant-openshift/action/install_openshift.rb]: you should be not setting this var at all, `osc login` will provide it.

Comment (soltysh) [inline lib/vagrant-openshift/action/install_openshift.rb]: :+1:

**Comment by mnagy** [comment]
PTAL

**Comment by mfojtik** [comment]
CC @liggitt

**Comment by liggitt** [comment]
How will users run commands as cluster admin?

**Comment by mfojtik** [comment]
@liggitt they can't with the current config anyway (the path is missing the directory).

**Comment by liggitt** [comment]
what directory? this is working today.. when you log in, you're the cluster admin...

**Comment by mfojtik** [comment]
@liggitt https://github.com/mnagy/vagrant-openshift/commit/3b74293a3d94f9b0a25fa05e75f5671f401092c2#diff-f88b5125e5d505d0cfc82bf22e527492L44 check the path, it starts with '/'

**Comment by liggitt** [comment]
yes... that's where the admin one gets generated...

**Comment by mnagy** [comment]
@liggitt why is that in `/`?

**Comment by liggitt** [comment]
that's the CWD the service starts from... etcd, config, and volumes dirs are there. Not saying that's the best place, that's just how it works today

**Comment by mnagy** [comment]
ok, my mistake, then. Closing.

**Pull request closed without merging**